documentation/modules/adding-source-provider.adoc

```diff
@@ -8,6 +8,8 @@ ifdef::vmware[]
 
 You can add a VMware vSphere source provider by using the {ocp} web console.
 
+In {project-short} versions 2.6 and later, you can also migrate VMs from standalone ESX/ESXi, or from ESX/ESXi servers that are managed by vCenter without going through vCenter.  This option is included in the procedure that follows.
```

Comment:
we don't mention vCenter before so it may not be so obvious to understand what "you can also" refers to
I'm also not sure if for "standalone ESX/ESXi" is a common term
how about:
```suggestion
In {project-short} versions 2.6 and later, in addition to migrations from vCenter, you can also migrate VMs from standalone ESX/ESXi, i.e., an ESX/ESXi server that is not managed by vCenter, or from ESX/ESXi servers that are managed by vCenter without going through vCenter.  This option is included in the procedure that follows.
```

Reply:
If "standalone ESX/ESXi" doesn't sound good to you, then I'll avoid it.  Also the sentence beginning "This option is included ..." is not needed -- since I mentioned the option, I'd better include it in the documentation :) My bad.